src/lib/variants.test.ts

```diff
@@ -372,14 +373,16 @@ describe('variant filters', () => {
     });
 
     it('should filter by articles viewed settings', () => {
+        const now = new Date('2020-03-31T12:30:00');
```

Comment:
This isn't necessarily related to this PR but I noticed we should be mocking the date object here somehow (like we do in the new filter) otherwise these tests will start failing down the road.